corehq/apps/userreports/tests/test_transforms.py

```diff
@@ -35,20 +36,20 @@ def test_int(self):
     def test_decimal(self):
         self.assertEqual('11', self.transform(11.23))
 
-    def test_strings(self):
-        tests = (
-            ('11', '11'),
-            ('11.23', '11'),
-            ('notanumber', 'notanumber'),
-            ('', ''),
-        )
-        for input, expected_result in tests:
-            self.assertEqual(expected_result, self.transform(input))
-
     def test_none(self):
         self.assertEqual(None, self.transform(None))
 
 
+@generate_cases((
+    ('11', '11'),
+    ('11.23', '11'),
+    ('notanumber', 'notanumber'),
+    ('', ''),
+), NumberFormatTransformTest)
+def test_number_format_transform_strings(self, input, expected_result):
+    self.assertEqual(expected_result, self.transform(input))
```

Comment:
I didn't try nose, but the others definitely do. If you declare it like this it just adds each case as a test to the `NumberFormatTransformTest`.